.gitignore

```diff
@@ -1,67 +1,45 @@
-# Compiled class file
-*.class
-*.balx
-# Log file
-*.log
-
-#conf file
-ballerina.conf
-Config.toml
-
-#lock file
-Ballerina.lock
+# Ignore Gradle project-specific cache directory
+.gradle
 
-# BlueJ files
-*.ctxt
+# Build Files
+target
+build
 
-# Mobile Tools for Java (J2ME)
-.mtj.tmp/
+# IDEA Files
+.idea/
+*.iml
+*.ipr
+*.iws
 
 # Package Files #
+!gradle/wrapper/gradle-wrapper.jar
 *.jar
-!gradle-wrapper/*.jar
 *.war
+*.nar
 *.ear
 *.zip
 *.tar.gz
 *.rar
 
-# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
-hs_err_pid*
-
-# Ignore everything in this directory
-target
-.classpath
-.settings
-.project
-*.iml
-*.iws
-*.ipr
-.idea
-.m2
-.vscode/
-*.ballerina/
-
-# Ignore ballerina files
-accessToken.bal
-temp.bal.ballerina/
-target/
-.DS_Store
-
-#Ignore emp-wrapper java classes
-**/target
-
-# Ignore Gradle project-specific cache directory
-.gradle
-
-# Build files
-**/build/*
+# MacOS
+*.DS_Store
 
-# Json files created inside resources folder. Refer exception below.
-**/resources/*.json
+# Ballerina
+velocity.log*
+*Ballerina.lock
 
-# Allow files in the bulk test folder
-!**/tests/resources/*
-
-# Ignore Gradle build output directory
-build
+# VSCode
+.vscode
+.project
+.settings/
+ballerina/.project
+ballerina/.settings/
+build-config/checkstyle/.classpath
+build-config/checkstyle/.project
+build-config/checkstyle/.settings/
+native/.classpath
+native/.project
+native/.settings/
+test-utils/.classpath
+test-utils/.project
+test-utils/.settings/
```

Comment:
Why do we need all these locations?

Reply:
Fixed with [2bfd8e2899585fa713ca440a10210084ab1282cf](https://github.com/ballerina-platform/module-ballerinax-azure-cosmosdb/pull/112/commits/2bfd8e2899585fa713ca440a10210084ab1282cf)